### PR TITLE
Refactor using Js classes

### PR DESCRIPTION
**Pull request by uuukiiiyooo**
This branch updates the previous feature (add and remove books from list/local storage) by using JavaScript classes.

-Refactored previos JavaScript code
-Updated styles to match medium wireframe

### CODE REVIEW

File: js/arrangeBooks.js [hunk 1]
```diff
@@ -1,53 +1,56 @@
...
-bookShelf.forEach((book) => displayBook(book));
-
-submitButton.addEventListener('click', () => {
-  const title = document.getElementById('bookTitle').value;
-  const author = document.getElementById('bookAuthor').value;
-  addBook(title, author);
-});
\ No newline at end of file
+const bookShelf = new BookShelf();
+console.log(bookShelf);
```
Comment (mckent05) [inline js/arrangeBooks.js]: - Please remove all `console.log` statement from your code, this will ensure that your code is readable and easy to maintain 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @christianonoh, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
- I get this error when i try running your project. Please fix this 🙏 
aerrerga

_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/arrangeBooks.js (hunk 1 above)
Comment (christianonoh) [reply js/arrangeBooks.js:56]: Hi Mckent,

I hope you're doing well. I'm writing to ask for some additional guidance on how to fix the "Missing script: start" error that I encountered when running the "npm run" command.

 I'm not sure what specifically needs to be added or changed in the file to fix this error.

Could you please provide me with more detailed guidance on how to fix this error? Any additional information you can provide would be greatly appreciated.

Thank you for your help!

**Comment by christianonoh** [comment]
Hi @mckent05,

I hope you're doing well. I'm writing to ask for some additional guidance on how to fix the "Missing script: start" error that I encountered when running the "npm run" command.

 I'm not sure what specifically needs to be added or changed in the file to fix this error.

Could you please provide me with more detailed guidance on how to fix this error? Any additional information you can provide would be greatly appreciated.

Thank you for your help!

**Review by Nemwel-Boniface — COMMENTED** [review]
## Hello @uuukiiiyooo  and @christianonoh  👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

I have inspected your project and realized that the CR @mckent05  might have made a mistake when requesting you to add the "Start" script. Currently your project does not make use of [Webpack](https://webpack.js.org/) and therefore your project cannot be started with ```npm start``` but rather from the liveserver.

For that, you can safely ignore that requested change by @mckent05 as that will only start applying to you when you set up and use Webpack in your next milestone (The TodoLIst projects)

However, [this requested](https://github.com/uuukiiiyooo/booksList/pull/2#discussion_r1144054214) change is valid and having console.logs in your codebase violates the JavaScript best practices. Kindly read through [this article](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) to familiarize yourself with the JavaScript best practices and ensure that you do not violate any of them in a future project.


Kindly remove all the console logs from your codebase and you can re-submit your project and the next reviewer will approve it. :rocket: 

In case anything is not 100% clear tag me @Nemwel-Boniface so that I get the notification.

Keep rocking 🚀!

**Review by frank1738 — APPROVED** [review]
Hi @uuukiiiyooo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ App works as expected 👍 


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**